### ISSUE TITLE
Accessibility: No clear mechanism to pause carousel

### ISSUE THREAD

**Issue by digilou** [FormidableLabs/nuka-carousel#782]
# Bugs and Questions

## Prerequisites

- [x] I've searched open [issues](https://github.com/FormidableLabs/nuka-carousel/issues) to make sure I'm not opening a duplicate issue
- [x] I have read through the docs before asking a question
- [x] I am using the latest version of nuka-carousel

## Describe Your Environment

- What version of nuka-carousel are you using? 4.7.5
- What version of React are you using? 16.13.1
- What browser are you using? Firefox 89.0

## Describe the Problem

When `autoplay=true`, a user can only pause the slide show by hovering over or clicking on it with the mouse pointer or advancing the slides with the prev/next buttons. This creates a barrier for people with cognitive (like ADHD), visual disabilities, and motor disabilities.

The best approach to provide a mechanism to pause/play the slides is to create a distinctly visible `<button>` on the carousel. Once activated, the button would either pause the carousel, or let it play again.

### Other requirements to be accessible
* This button should have an accessible name that describes its purpose (dynamic name to update when state changes). If the button includes an SVG icon, the icon should include that accessible name (button's purpose), or the icon should be `aria-hidden` and text should be available (visible or visually hidden).
* This button should be keyboard accessible (using `button` helps with that). 
* This button should receive a visible focus when a user tabs via keyboard to it.
* Tabbing to this button should be in a logical order that matches the visual logical order. For example, help the user get to this button first, then slide advance buttons, then slide nav dots.

Loosely relates to #499. 

## Additional Information
This basic functionality improves the accessibility of this plugin and conforms to WCAG 2 Level A, [success criterion 2.2.2: Pause, Stop, Hide](https://www.w3.org/WAI/WCAG21/quickref/#pause-stop-hide).

**Comment by sarmeyer** [comment]
Hi @digilou thank you so much for writing in with this request. It is already possible to do this with the `autoplay` prop and the current demo has an example of controlling whether the carousel is "paused" or "playing" using react state. You can style and customize something similar in your implementation of Nuka Carousel that will fit your needs and a11y requirements

```
<button onClick={() => setAutoplay((prevAutoPlay) => !prevAutoPlay)}>
    Toggle Autoplay {autoplay === true ? 'Off' : 'On'}
</button>

**Comment by digilou** [comment]
Thanks for clearing that up for me @sarmeyer.
